indico/modules/events/management/controllers.py

```diff
@@ -178,3 +178,13 @@ def _update_session_coordinator_privs(self, form):
             log_msg = 'Session coordinator privilege changed to {}: {}'.format(form.data[priv_field],
                                                                                COORDINATOR_PRIV_TITLES[priv_field])
             self.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Protection', log_msg, session.user)
+
+
+class RHMoveEvent(RHConferenceModifBase):
+    """Move event to a different category"""
+
+    def _process(self):
+        category = Category.find(id=request.json.get('category_id')).first_or_404()
```

Comment:
Any reason for using `request.json.get()`? Not having the value doesn't make much sense so getting a KeyError if it's missing is probably the better choice.